guides/common/modules/proc_creating-a-user.adoc

```diff
@@ -44,4 +49,6 @@ The `--auth-source-id 1` setting means that the user is authenticated internally
 Add the `--admin` option to grant administrator privileges to the user.
 Specifying organization IDs is not required, you can modify the user details later using the `update` subcommand.
 
-For more information about user related subcommands, enter `hammer user --help`.
+.Additional resources
+
+* For more information about subcommands related to user accounts, enter `hammer user --help`.
```

Comment:
This section is on creating users so I think that being more specific here is fine.
```suggestion
* For more information about creating user account with `hammer`, enter `hammer user create --help`.
```

Reply:
Just a bit different wording:
```suggestion
* For more information about creating user accounts by using Hammer, enter `hammer user create --help`.
```